src/openrct2/OpenRCT2.h

```diff
@@ -22,18 +22,20 @@ enum class StartupAction
     Edit
 };
 
-enum
+enum class ScreenMode
```

Comment:
Previously, we've renamed the 'screen' terminology to 'scene' to better align with game engines. Ultimately, the aim was to entirely do away with these 'flags' and instead query a 'scene manager', but clearly we haven't gotten to that point yet.

In the mean time, I think it would be better for the type and associated global to align with the 'scene' terminology. That would also prevent potential confusion with 'screen modes' as in resolutions. As for the name, `SceneMode` is fine, but perhaps `LegacyScene` would be more future proof? Just a thought.